Postlaunch Frontend 1

sidramatik opened the pull request:
Will likely require one additional round of dev but this is to check the updates page templates.

careduz commented on wp-open-data-toronto/style.css:
```diff
@@ -3,7 +3,7 @@ Theme Name:      WP Open Data Toronto
 Author:          Sidra Mahmood, Yizhao Tan, Carlos Hernandez
 Template:        wp-bootstrap-starter
-Version:         1.2.0
+Version:         2.0.1
```
Does version refer to the overall WordPress theme or the stylesheet specifically?

If the former, this should be 2.1.0 although I don't think this is the best place to capture this because releases could involve non-styling changes and it would be too easy to go out of date.

If the latter, need to keep track of stylesheet versioning independently though I don't see much value in that.

What do you think?

Regardless, since we are here, just noticed we should update the template name to `wp-open-data-toronto`

sidramatik replied:
Removed

careduz commented:
Could you please add the list of changes to the Pull Request? Understand it won't be comprehensive, just want to ensure we have a change log in place.

sidramatik commented on wp-open-data-toronto/homepage.php:
```diff
@@ -85,12 +85,12 @@
           <div class="col-md-4">
             
-            <h2><a href="/budgetpedia-humanizing-city-budgets/">Showcase: Budgetpedia</a></h2>
+            <h2><a href="/torontos-open-data-initiative-2018-in-review/">2018 In Review</a></h2>
```
This is resolved :)

sidramatik commented:
List of changes in this pull request:

- Updated stylesheet to remove version and edited template name to avoid confusion
- Updated homepage so that featured posts are dynamic and loaded from the WP backend, as opposed to within the source code itself (Please note: you MUST create these fields in contrib in Toolset Types for them to appear in the front-end).
- Updated blog template to display all published posts regardless of category
- Additional improvements for mobile styling, such as standardized padding and font scaling
- Additional accessibility improvements, such as focus states 
- Added new header and footer styling as per strategic communications requirements
